scripts/apidoc/utils.ts

```diff
@@ -105,11 +106,29 @@ export function extractRawExamples(signature?: SignatureReflection): string[] {
   );
 }
 
+/**
+ * Extracts the all see also references from the jsdocs separately.
+ *
+ * @param signature The signature to extract the see also references from.
+ */
+export function extractSeeAlsos(signature?: SignatureReflection): string[] {
+  return extractTagContent('@see', signature, (tag) => {
+    const content = tag.content;
+    if (content.length === 1) {
+      return joinTagContent(tag);
+    } else {
+      return tag.content
+        .filter((_, index) => index % 3 === 1) // ['-', 'content', '\n']
+        .map((part) => part.text);
+    }
```

Comment:
You can omit this `else` and just return as you have a return in your `if`

```suggestion
    }
    return tag.content
      .filter((_, index) => index % 3 === 1) // ['-', 'content', '\n']
      .map((part) => part.text);
```